pkgs/top-level/all-packages.nix

```diff
@@ -25777,7 +25777,9 @@ with pkgs;
 
   systemd-journal2gelf = callPackage ../tools/system/systemd-journal2gelf { };
 
-  tailscale = callPackage ../servers/tailscale { };
+  tailscale = callPackage ../servers/tailscale {
+    buildGoModule = buildGo123Module;
+  };
```

Comment:
It seems we may also need to override it for:
- `tailscale-gitops-pusher`
- `tailscale-nginx-auth`